Test/Tasty/Runners/AntXML.hs

```diff
@@ -130,11 +131,14 @@ antXMLRunner = Tasty.TestReporter optionDescription runner
           Const summary <$ State.modify (+ 1)
 
         runGroup groupName children = Tasty.Traversal $ Functor.Compose $ do
-          Const soFar <- Reader.withReaderT (++ [groupName]) $ Functor.getCompose $ Tasty.getTraversal children
+          Const soFar <- Reader.local (groupName :) $ Functor.getCompose $ Tasty.getTraversal children
```

Comment:
nit:
pre-append groupName then `reverse` seems a bit waist rather than append to end.